modules/understanding-upgrade-channels.adoc

```diff
@@ -89,7 +89,7 @@ First, select the minor version you want for your cluster upgrade. Selecting a c
 
```

Comment:
Did you want to change update in the first sentence in the paragraph above? ("cluster upgrade") (probably a reason not to, but I thought I'd ask)

Reply:
No reason, I seemed to have missed that instance somehow. Thanks for catching this!